blueprints/util/files/__root__/utils/__name__.coffee

```diff
@@ -1,4 +1,2 @@
-<%= camelizedModuleName %> = () ->
+export default () ->
```

Comment:
In [the original blueprint](https://github.com/ember-cli/ember-cli-legacy-blueprints/blob/master/blueprints/util/files/__root__/utils/__name__.js) this is a named function, but I haven't found a way to export a named function with the CoffeeScript syntax, so I'm not sure if it can be done.